client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -8,14 +8,14 @@ import { noop, get } from 'lodash';
 /**
  * Internal dependencies
  */
+import makeJsonSchemaParser from 'lib/make-json-schema-parser';
+import schema from './schema.json';
+import { clearJITM, insertJITM } from 'state/jitm/actions';
 import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { getSelectedSiteId } from 'state/ui/selectors';
```

Comment:
Do you sort the imports with some algorithm or editor plugin? Just curious :wink:

Reply:
vim `:sort i` (case-insensitive sort) usually with visual selection.

I manually sort multi-line imports after single-line 🤷‍♂️